python/cudf/cudf/core/frame.py

```diff
@@ -1957,8 +1957,8 @@ def __dask_tokenize__(self):
         from dask.base import normalize_token
 
         return [
-            type(self),
-            normalize_token(self._dtypes),
+            str(type(self)),
+            str(self._dtypes),
```

Comment:
**Note**: After some recent tokenization fixes in dask, we now need to "stringify" this information in order for the token to be deterministic.

Reply:
Okay, looks like tokenizing types is not the problem, so I will roll back part of this change. As far as I can tell, https://github.com/dask/dask/pull/10904 tweaked the behavior of nested `dict` normalization, and what we really want is to avoid normalizing the `_dtypes` within `__dask_tokenize__` like this. Can't say I completely understand what was failing in `test_dispatch.py::test_deterministic_tokenize` without this "fix".